x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -162,24 +108,17 @@ export function OverviewPage({ routeParams }: Props) {
         </EuiFlexItem>
 
         {/* Alert section */}
-        {!!alerts.length && (
-          <EuiFlexItem grow={3}>
-            <AlertsSection alerts={alerts} />
-          </EuiFlexItem>
-        )}
+        <AlertsSection setAlertEmptySection={setAlertEmptySection} />
 
         {/* Resources section */}
         <EuiFlexItem grow={1}>
           <EuiFlexGroup direction="column">
             <EuiFlexItem grow={false}>
               <Resources />
             </EuiFlexItem>
-
-            {!!newsFeed?.items?.length && (
-              <EuiFlexItem grow={false}>
-                <NewsFeed items={newsFeed.items.slice(0, 5)} />
-              </EuiFlexItem>
-            )}
+            <EuiFlexItem grow={false}>
+              <NewsFeed />
+            </EuiFlexItem>
```

Comment:
One issue I see with this approach is that we are rendering a EUIFlexItem even if there is no news. What happens when we add an item below? Do we get things like double padding/margin?